README.md

```diff
@@ -31,7 +31,43 @@ If you are a stack owner and need to request an urgent refresh of <https://regis
 
 To build this devfile registry into a container image run `bash .ci/build.sh`. A container image will be built using the [devfile registry build tools](https://github.com/devfile/registry-support/tree/master/build-tools). By default these scripts will use `docker`, if you want to use `podman` you should first run `export USE_PODMAN=true` before executing the build script.
 
-From there, push the container image to a container registry of your choice and deploy using one of the methods outlined [here](https://github.com/devfile/registry-support#deploy).
+From there, push the container image to a container registry of your choice and deploy using a [devfile adopted devtool](https://devfile.io/docs/2.2.0/developing-with-devfiles#tools-that-provide-devfile-support) or one of the methods outlined [here](https://github.com/devfile/registry-support#deploy).
+
+## Devfile Deployments
+
+### Prerequisites
+
+- Docker 17.06 or later
+- Git
+- Kubernetes or Red Hat OpenShift
+- odo v3.15.0+
+
+### Deploying
+
+The following can be used to build and deploy a devfile registry locally:
+
+odo V3: 
+
+```sh
+odo deploy --var indexImageName=quay.io/<user>/devfile-index --var indexImageTag=<tag>
```

Comment:
Should we also add here the `hostName` variable and suggest a custom host name for local deployments?

Reply:
Should be resolved in https://github.com/devfile/registry/pull/256/commits/4d7dc5e5b31c6ca09d15ddee9712f97a70f2ba4e.